scalableminds/webknossos
# Call Long-Running Jobs from webKnossos

Run long-running jobs converting tiff stacks to wkw, via https://github.com/scalableminds/webknossos-worker

### Steps to test:

- set `jobsEnabled = true` in `features` block in `conf/application.conf`
- flower and celery need to run and have the `tiff_cubing` task registered, compare https://github.com/scalableminds/webknossos-worker
- During dataset upload, check “needs conversion” for zipped tif stack dataset.
- After upload you should be redirected to new page Administration→Jobs, where cubing job should be listed. Page should auto-refresh.
- After successful cubing job, a view link to the now usable dataset should be shown.

- This uses new backend api routes `/jobs/run/cubing`, `/jobs`, `/jobs/:id/status` 

### Issues:
- fixes #4643 
- fixes #4337 

------
